apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/LoggerOptions.cs

```diff
@@ -12,40 +12,50 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using Google.Api;
 using Google.Api.Gax;
 using Google.Cloud.Diagnostics.Common;
 using Microsoft.Extensions.Logging;
 
 namespace Google.Cloud.Diagnostics.AspNetCore
 {
-    // TODO(talarico): Add option for logging to other resources and organizations.
-
     /// <summary>
     /// Options for a <see cref="GoogleLogger"/>.
     /// </summary>
     public sealed class LoggerOptions
     {
+        /// <summary>The global resource.</summary>
+        internal static readonly MonitoredResource s_globalResource = new MonitoredResource { Type = "global" };
```

Comment:
Again, probably doesn't matter too much, but I think the s_ naming is supposed to be for private fields only. Use `GlobalResource` here.